_layouts/default.html

```diff
@@ -13,7 +13,7 @@
   {%comment%}
   **TODO** Implement RSS feed once posts implemented
   {%endcomment%}
-  <link rel="alternate" type="application/rss+xml" title="Mike Down South &raquo; Feed" href="{{site.baseurl}}/feed/" />
+  <!--<link rel="alternate" type="application/rss+xml" title="Mike Down South &raquo; Feed" href="{{site.baseurl}}/feed/" />-->
```

Comment:
That is actually the blog's RSS feed.  I haven't created that piece yet and should've mentioned that on my todo list I suppose, but it should be pretty easy to do so thought you might still want an RSS feed?  I dunno, is that important to you or do you want to remove it entirely?  Either way is okay on my end...

I did just comment it out in the short term until I had a chance to re-create it to match the WP rss feed for the blog, because the HTML Proofer was seeing it and looking for the associated page but couldn't find it!  So, kinda cool that the test caught it, but again just figured comment it out in the short term...

Up to you how you want to handle, just lemme know!